src/main/java/seedu/ccacommander/logic/parser/CcaCommanderParser.java

```diff
@@ -65,8 +66,11 @@ public Command parseCommand(String userInput) throws ParseException {
         case CreateEventCommand.COMMAND_WORD:
             return new CreateEventCommandParser().parse(arguments);
 
-        case EditCommand.COMMAND_WORD:
-            return new EditCommandParser().parse(arguments);
+        case EditMemberCommand.COMMAND_WORD:
+            return new EditMemberCommandParser().parse(arguments);
+
+        case EditEventCommand.COMMAND_WORD:
+            return new EditEventCommandParser().parse(arguments);
```

Comment:
As codecov has flagged it already, do you want to a relevant test?

Reply:
Noted! Have added a relevant test.